integrations/visual_studio/msbuild.rst

```diff
@@ -0,0 +1,98 @@
+.. _msbuild_integration:
+
+|visual_logo| MSBuild Integration
+=================================
```

Comment:
This section needs a working example in the "examples" repo.

Reply:
The examples in the "Creating and reusing packages with Visual Studio" chapter are already using MSBuild, but that chapter comes later:

![image](https://user-images.githubusercontent.com/589897/143231104-7490e547-9f08-4dce-b528-b1ab933b5049.png)

Maybe we should link that examples here somehow?